### PR TITLE
Energy page edits

### PR DESCRIPTION
**Pull request by rkelson**
.

### CODE REVIEW

**Comment by rkelson** [comment]
I found somethings that need fix'n

**Comment by rkelson** [comment]
the last commit was the only issue I found

File: data/energy/energy-generation/index.html [hunk 1]
```diff
@@ -8,9 +8,9 @@
 title: Energy Generation
```
Comment (steveoh) [inline data/energy/energy-generation/index.html:8]: **nitpick**: remove title
```suggestion

```

File: data/energy/renewable-energy/index.html [hunk 2]
```diff
@@ -93,9 +80,10 @@ <h3 id="WindEnergy">Wind Energy</h3>
     <p>This dataset depicts renewable wind resources identified by the Utah Renewable Energy Zone (UREZ) task force and contains the site name, wind potential, location comments and more.</p>
     <h5>Related Resources</h5>
     <ul class="dotless">
-      <li><a href="https://drive.google.com/drive/folders/0ByStJjVZ7c7mcmY2ZGphbUtMQTA">UREZ Phase1 Transmission Lines</a></li>
-      <li><a href="https://drive.google.com/drive/folders/0ByStJjVZ7c7md2R5aXNPYm5HcGM">UREZ Phaze2 Conceptual Transmission Lines</a></li>
-      <li><a href="https://drive.google.com/drive/folders/0ByStJjVZ7c7maGt1TlljVzRwckE">UREZ Phaze2 Zone Boundaries</a></li>
+      <li><a href="https://opendata.gis.utah.gov/datasets/utah-urez-phase-1-transmission">UREZ Phase1 Transmission Lines</a></li>
+      <li><a href="https://opendata.gis.utah.gov/datasets/utah-urez-phase-2-conceptual-transmission">UREZ Phaze2 Conceptual Transmission Lines</a></li>
+      <li><a href="https://opendata.gis.utah.gov/datasets/utah-urez-phase-2-zone-boundariesE">UREZ Phaze2 Zone Boundaries</a></li>
```
Comment (steveoh) [inline data/energy/renewable-energy/index.html:85]: **issue**: fix link
```suggestion
      <li><a href="https://opendata.gis.utah.gov/datasets/utah-urez-phase-2-zone-boundaries">UREZ Phaze2 Zone Boundaries</a></li>
```

**Review by steveoh — APPROVED** [review]

**Pull request merged**